src/features/game/expansion/components/travel/IslandList.tsx

```diff
@@ -129,9 +125,14 @@ export const IslandList: React.FC<IslandListProps> = ({
   bumpkin,
   showVisitList,
   inventory,
+  travelAllowed,
 }) => {
   const { authService } = useContext(Auth.Context);
-  const [authState, send] = useActor(authService);
+  const farmId = useSelector(authService, (state) => state.context.farmId);
```

Comment:
It is best practice to define the `useSelector` callbacks outside of the component as mentioned in the [docs](https://xstate.js.org/docs/packages/xstate-react/#api). This way the arrow function is not redefined on each rerender.

Would we be able to move these outside?

Reply:
@spencerdezartsmith, Done.